crmsh/sbd.py

```diff
@@ -212,6 +211,30 @@ def _get_sbd_device_from_config(self):
         else:
             return None
 
+    def _is_delay_start(self):
+        """
+        Check if SBD_DELAY_START is yes
+        """
+        res = SBDManager.get_sbd_value_from_config("SBD_DELAY_START")
+        return True if res and res == "yes" else False
+
+    @staticmethod
+    def get_sbd_watchdog_timeout():
+        """
+        Get SBD_WATCHDOG_TIMEOUT from /etc/sysconfig/sbd
+        """
+        res = SBDManager.get_sbd_value_from_config("SBD_WATCHDOG_TIMEOUT")
+        if not res:
+            raise ValueError("Cannot get the value of SBD_WATCHDOG_TIMEOUT")
+        return int(res)
+
+    @staticmethod
+    def get_sbd_systemd_timeout():
+        """
+        Systemd start timeout for sbd.service
+        """
+        return int(SBDManager.get_sbd_watchdog_timeout() * 2 * 1.2)
```

Comment:
For more robustness and correctness, how about,
```
if self.diskless_sbd:
    return int(SBDManager.get_sbd_watchdog_timeout() * 2 * 1.2)
else:
    return int(SBDManager.get_sbd_msgwait() * 1.2)
```

Reply:
Changed
Thanks!